src/core_modules/capture-core/components/Pages/MainPage/MainPage.constants.js

```diff
@@ -2,5 +2,6 @@ export const MainPageStatuses = Object.freeze({
     DEFAULT: 'DEFAULT',
     WITHOUT_ORG_UNIT_SELECTED: 'WITHOUT_ORG_UNIT_SELECTED',
     WITHOUT_PROGRAM_CATEGORY_SELECTED: 'WITHOUT_PROGRAM_CATEGORY_SELECTED',
+    CATEGORY_OPTION_INVALID_FOR_ORG_UNIT: 'CATEGORY_OPTION_INVALID_FOR_ORG_UNIT',
```

Comment:
Is it possible to use the same const string for both the Main page and the New page status, either `CATEGORY_OPTION_INVALID_FOR_ORG_UNIT` or `INVALID_CATEGORY_OPTION_SELECTED`? Thanks!